src/js/util.js

```diff
@@ -41,7 +41,8 @@ var Util;
             ESCAPE: 27,
             SPACE: 32,
             DELETE: 46,
-            K: 75 // K keycode, and not k
+            K: 75, // K keycode, and not k
+            ENTER_OF_EMACS: 77
```

Comment:
77 is actually the key code for the 'M' key on the keyboard, so would you rename this to just `keyCode.M`?